README.md

```diff
@@ -13,26 +29,39 @@ $ make
 $ sudo make install
 ```
 
+### OS X specific steps
+
+Several packages are required to complete this build, including `Python` and `pkg-config`.  To install all the dependencies on OS X using `brew`:
+
+```
+brew install cmake
+brew install protobuf
+brew install grpc
+brew install python
+brew install pkg-config
```

Comment:
Where does the python dependency come up?

Also, I think OS X ships with python installed by default.

Reply:
I'll test again. I think I had to update something.

Reply:
Confirmed: Python was installed as a result of installing protobuf. The three I need on OS X to install myself are protobuf, grpc, and pkg-config.